workflow-v2.1.0-e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0 e2e Tests (parallel)
+
+End-to-end tests for the Deis Workflow open source PaaS, executed in parallel.
+
+NOTE: This assumes a fresh `helmc install` of Deis v2. The test suite creates
+an initial user with admin privileges (who is deleted when tests complete).
+See https://github.com/deis/workflow-e2e for more detail.
+
+```console
+$ helmc uninstall -n deis -y workflow-e2e ; helmc install workflow-e2e
+$ kubectl --namespace=deis logs -f workflow-e2e tests
```

Comment:
I'd noticed that. But isn't the pod name configurable now? Does that make it hard to accurately put the command here that will allow someone to view the tests in-flight?